lib/rmt/import_script.rb

```diff
@@ -0,0 +1,161 @@
+require 'csv'
+require 'ostruct'
+
+
+class ImportScript
+  attr_accessor :config
+
+  class ExitScript < StandardError
```

Comment:
`ImportScript::ExitScript` doesn't sound very nice. Rename to `ImportException`, perhaps?